.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Question
+about: Ask a question about this STL implementation
+title: ""
+labels: question
+assignees: ''
+
+---
+
+Instructions
+============
+Here, you can ask a question about this STL implementation,
+and a maintainer will answer.
+
+On-Topic Examples
+-----------------
+* What is this code in the STL doing? You can link to the relevant code:
+https://help.github.com/en/github/managing-your-work-on-github/creating-a-permanent-link-to-a-code-snippet
+* What are the preferred conventions for writing something?
+* What are the maintainers planning to do in the future?
+* Would the maintainers be interested in specific enhancements?
+
+Off-Topic Examples
+------------------
+* Questions about non-STL components, such as the compiler, the CRT,
+the Windows API, the Visual Studio IDE, etc.
+* Questions about how to use the STL or C++ in general. For such questions,
+consider https://stackoverflow.com or https://www.reddit.com/r/cpp_questions/ .
+* Questions about whether you've encountered a bug in the STL. Instead of this
+Question issue template, please use the Bug Report issue template, because

```

Comment:
I would change "issue template" to just "template". "Question template" and "Bug Report template" sound better.

Reply:
Great idea, done!